### PR TITLE
{vis}[foss/2023a] empanada-napari v1.1.0, empanada-dl v0.1.7, connected-components-3d v3.14.1, Albumentations v1.4.0

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/l/libavif/libavif-1.0.4-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('NASM', '2.16.01'),
+    ('Meson', '1.1.1'),
+    ('Ninja', '1.11.1'),
+    ('Rust', '1.75.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/l/libavif/libavif-1.0.4-GCCcore-12.3.0.eb:29]: @pavelToman Is `Rust` 1.70.0 sufficiently recent? If so, let's stick to that.

Alternatively (and maybe better), `Rust` is probably a build-only dependency for this, and same goes for Meson + Ninja I think...

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/l/libavif/libavif-1.0.4-GCCcore-12.3.0.eb (hunk 1 above)
Comment (pavelToman) [reply easybuild/easyconfigs/l/libavif/libavif-1.0.4-GCCcore-12.3.0.eb]: I moved all the deps to builddeps - seems the `NASM` could be there either - it is `General-purpose x86 assembler` and in some easyconfigs is in builddeps.

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 16 out of 16 (16 easyconfigs in total)
node3139.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/a89c4583a6d5dd6be93121d27135f399 for a full test report.

**Review by boegel — DISMISSED** [review]

**Comment by verdurin** [comment]
@pavelToman unfortunately I missed this PR when I merged https://github.com/easybuilders/easybuild-easyconfigs/pull/20663, hence the conflicts.

Could you remove `imagecodecs` and update your ECs to use the newer `imagecodecs`?
Perhaps some other merged PRs are implicated here too, and you might need to do the same for them.

**Comment by boegel** [comment]
@verdurin There a no merged easyconfig that use a `2023a` toolchain and depend on `imagecodecs`, so "fallout" is limited to this PR it seems.

**Comment by pavelToman** [comment]
- https://github.com/easybuilders/easybuild-easyconfigs/pull/20636

**Comment by pavelToman** [comment]
@verdurin @boegel Could you help me with `libwebp`? How to get rid of its changes in the PR?

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
I've re-installed the missing `Arrow/14.0.1-gfbf-2023a` module on `jsc-zen3`, so let's try again...

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3107.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/b8bfbdcc8b7e26378617895791c94cce for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**